src/internal_modules/roc_pipeline/receiver_session.h

```diff
@@ -92,6 +94,11 @@ class ReceiverSession : public core::RefCounted<ReceiverSession, core::ArenaAllo
     //! Handle estimated link metrics.
     void add_link_metrics(const rtcp::LinkMetrics& metrics);
 
+    //! Gets recent latency statistics.
+    //! @returns
+    //! True if statistics is valid.
+    bool stats(Stats& stats) const;
```

Comment:
Let's simplify it:

```
Stats stats() const;
```

I think there is no need to return bool. Instead, if some of the stats are not available for whatever reason (e.g. there is no latency info), they should be set to zero.